package.json

```diff
@@ -102,29 +43,17 @@
     "eslint-plugin-jsdoc": "^36.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.4.1",
-    "ethjs-provider-http": "^0.1.6",
-    "jest": "^26.4.2",
-    "jest-environment-jsdom": "^25.0.0",
```

Comment:
Hmm. Was this not needed? I wonder why we added this

Reply:
Hmm. I never saw this issue for some reason.

Reply:
It was [closed as fixed in February](https://github.com/facebook/jest/issues/7780#issuecomment-1046367215), though it's unclear which Jest version it was fixed in. Maybe it's fixed in the version we're using.